SIL.WritingSystems.Tests/SldrTests.cs

```diff
@@ -493,9 +471,7 @@ public void LanguageTags_NoCachedLangTagsSldrOffline_UseEmbeddedLangTags()
 			{
 				var langTagsPath = Path.Combine(Sldr.SldrCachePath, "langtags.json");
 				Assert.That(File.Exists(langTagsPath), Is.False);
-
 				Assert.That(Sldr.LanguageTags, Is.Not.Empty);
-				Assert.That(File.Exists(langTagsPath), Is.False);
```

Comment:
Why remove this line?

Reply:
Yes, but I think it makes sense to have it in both places: the first time verifies the setup, i.e. that we start in the state we expect with the file being absent. The second time it verifies that calling `Sldr.LanguageTags` did not create the file.

But I agree, it looks confusing and could use some comments to make it clearer...